src/routes/admin/index.js

```diff
@@ -3,7 +3,30 @@ const router = express.Router()
 const Book = require( '../../database/queries/book')
 
 router.get( '/', function( request, response ) {
-  response.send('helo im admin')
+  response.render('admin/add_book')
+})
+
+router.post( '/add', function( request, response) {
+  const {title, price, image_url, page_count, description} = request.body
+  const book_info = {title, price, image_url, page_count, description}
+  const authors = request.body.author_name
+  const genres = request.body.genre_name
+  console.log('typeof request.body, request.body:', typeof request.body, request.body)
+
+
+
+  let new_book_info = Object.keys( book_info ).reduce( (memo, key) => {
```

Comment:
I would use `const` unless you intend to mutate the result (not that it really matters for an object, but the intent is clearer with const)